.drone.star

```diff
@@ -1183,11 +1175,7 @@ def e2eTests(ctx):
                          "pnpm test:e2e:cucumber %s" % " ".join(params["featurePaths"]),
                      ],
                  }] + \
-                 uploadTracingResult(ctx) + \
-                 publishTracingResult(ctx, "e2e-tests %s" % server)
-        if (params["earlyFail"]):
-            steps += buildGithubCommentForBuildStopped("e2e-ocis" if server.startswith("oCIS") else "e2e-oc10")
-        steps += githubComment("e2e-tests", server)
+                 uploadTracingResult(ctx)
```

Comment:
Do we also remove the `uploadTracingResult` ? or not needed?

cc @saw-jan @phil-davis @ScharfViktor

Reply:
we may want e2e tracings for debugging

Reply:
I commented here: https://github.com/owncloud/web/pull/9392#pullrequestreview-1525685614